probe-rs/src/session.rs

```diff
@@ -400,6 +461,8 @@ impl Session {
 // This test ensures that [Session] is fully [Send] + [Sync].
 static_assertions::assert_impl_all!(Session: Send);
 
+/*
+ // TODO tiwalun: Enable again, after rework of Session::new is done.
```

Comment:
Could this be enabled again? Not sure why I had to disable this :confused:

Reply:
To be honest I have no clue why it was disabled but imma try and reenable it!